go/border/scheduler.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import "github.com/scionproto/scion/go/lib/log"
+
```

Comment:
This should most likely be an interface and the things below implementations of it.